### PR TITLE
add instance piped.moomoo.me

### PR DESCRIPTION
**Pull request by lucamp**
this instance is working really well in Italy

also removed last / from line 98:
"https://piped.notyourcomputer.net/"

### CODE REVIEW

**Comment by EdwardLangdon** [comment]
"https://piped.notyourcomputer.net/" it is working well. And I can access it. You haven't removed the Instance yet from the code. So, Don't do it.

The Reason, we haven't added moomoo Instance frontend URL. It is only available to use via API most of the time. So, At the time of adding Instances moomoo.me wasn't working. So, We haven't added. Now, The frontend seems to be online. In my POV, The Frontend of Moomoo is known for many Downtimes.

We shall see whether we will merge this PR or not after speaking with @ManeraKai

**Comment by lucamp** [comment]
@EdwardLangdon I didn't state that I will remove notyourcomputer.net. Please read again my first message.

Notmycomputer is localed in US and has no CDN, from Italy it's slower than moomoo.me (or silkky.cloud)

I have no idea when you checked moomoo.me, from my Italy no issue in 2 weeks and one of the best ping

**Comment by EdwardLangdon** [comment]
You can see my PR here - https://github.com/libredirect/libredirect/pull/132. I checked it about 11 days ago. At the time, I cannot access it. I am from Swiss.

>Please read again my first message.
Sorry, I didn't understand what you meant at the time. I get it now.

**Comment by EdwardLangdon** [comment]
As of now, I cannot access piped.notyourcomputer.net. Are you able to access it ?

For moomoo.me, I am talking about availability all the time. Let's wait for a few days. If you can, Log that, from now, Day 1 to Day 10 from Now on. How often you can access Moomoo Instance.

We want to be sure that the addition must be worth the thing. Not the one unavailable after running it.

**Comment by lucamp** [comment]
> As of now, I cannot access piped.notyourcomputer.net. Are you able to access it ?

Up and running. Maybe a DNS issue? 

> For moomoo.me, I am talking about availability all the time. Let's wait for a few days. If you can, Log that, from now, Day 1 to Day 10 from Now on. How often you can access Moomoo Instance.

No problem

**Comment by EdwardLangdon** [comment]
> Up and running. Maybe a DNS issue?

You are right.

**Comment by ManeraKai** [comment]
5 days have passed. I'll merge it.

**Comment by ManeraKai** [comment]
For me it it gives a 502 Error from Cloudflare even through Tor.

**Comment by lucamp** [comment]
> For me it it gives a 502 Error from Cloudflare even through Tor.

Piped.moomoo.me is down since few hours ago, [https://status.moomoo.me](https://status.moomoo.me)

There have been other downs that as a end user I did not noticed and at this point is not reliable

**Comment by EdwardLangdon** [comment]
@lucamp Are you making logs as I requested ?

**Comment by lucamp** [comment]
> @lucamp Are you making logs as I requested ?

As I wrote in my previous message, piped.moomoo.me was down. And at [https://status.moomoo.me/](https://status.moomoo.me/) you can check the previous incident, such as the one you reported.

There is no point to keep logging something that breaks every 2 weeks, and to include it in the list as a default instance. Just reject this pull

**Review by EdwardLangdon — CHANGES_REQUESTED** [review]
All Good. Just rearrange the list according to the official list after this PR is merged - https://github.com/libredirect/libredirect/pull/179

**Comment by EdwardLangdon** [comment]
@lucamp according to the Status page it is down only at certain Intervals and seems to be up. I think we can add this Instance, If it is ever offline, It will autmatically redirect to another one which is online.

File: src/assets/javascripts/helpers/youtube/youtube.js
Comment (EdwardLangdon) [inline src/assets/javascripts/helpers/youtube/youtube.js]: Add `,` at the end of line 98.

**Review by EdwardLangdon — CHANGES_REQUESTED** [review]

**Pull request merged**
